Task4/FactoryPattern.java

```diff
@@ -0,0 +1,55 @@
+package Task4;
+
+import java.util.Scanner;
+
+//Client Specifying what type of object he wants to the factory
+public class FactoryPattern {
+	public static void main(String[] args) throws Exception {
+		ShoeFactory factory = new ShoeFactory();
+		Scanner inp = new Scanner(System.in);
+		System.out.println("Enter the type of sneaker");
+		Sneakers shoe = factory.createSneaker(inp.next());
```

Comment:
Can you add error handling here? `createSneaker()` might throw an error but it's not handled here.